app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int, average_rating: float,
+            count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            income += self.wash_single_car(car)
+        return round(income, 1)
+
+    def wash_single_car(self, car: Car) -> float:
+        if self.clean_power >= car.clean_mark:
+            income = round(self.calculate_washing_price(car), 1)
+            car.clean_mark = self.clean_power
+            return income
+        else:
+            return 0
```

Comment:
Refactor it without using `else`
```suggestion
    def wash_single_car(self, car: Car) -> float:
        if self.clean_power >= car.clean_mark:
            income = round(self.calculate_washing_price(car), 1)
            car.clean_mark = self.clean_power
            return income
        return 0
```